mflowgen/common/fc-netlist-fixing/outputs/netlist-fixing.tcl

```diff
@@ -0,0 +1,57 @@
+#netlist fixing for pad ring
+addNet ESD_0 -ground -physical
+addNet ESD_1 -ground -physical
+addNet ESD_2 -ground -physical
+addNet ESD_3 -ground -physical
+
+addNet POC_0 -ground -physical
+addNet POC_1 -ground -physical
+
+addNet AVDD -power -physical
+addNet AVSS -ground -physical
+addNet CVDD -power -physical
+addNet CVSS -ground -physical
```

Comment:
These (AVDD, CVDD...) are for analog block, yes? Do we not also need nets TACVSS, TACVDD?

Reply:
I see; maybe; the globalNetConnect commands below mean that we connect e.g. TACVSS pins either to net CVSS or AVSS depending on the analog IO pad.